Jenkinsfile

```diff
@@ -37,18 +37,19 @@ node('vetsgov-general-purpose') {
           if (!shouldBuild) { return }
 
           try {
+            // try to build using fresh drupal content
             commonStages.build(ref, dockerContainer, assetSource, envName, false, contentOnlyBuild)
             envUsedCache[envName] = false
           } catch (error) {
-            if (!contentOnlyBuild) {
+            try {
+              // try to build again using the cached drupal content
               dockerContainer.inside(DOCKER_ARGS) {
                 sh "cd /application && node script/drupal-aws-cache.js --fetch --buildtype=${envName}"
               }
               commonStages.build(ref, dockerContainer, assetSource, envName, true, contentOnlyBuild)
               envUsedCache[envName] = true
-            } else {
-              commonStages.build(ref, dockerContainer, assetSource, envName, false, contentOnlyBuild)
-              envUsedCache[envName] = false
+            } catch (err) {
+              throw err
```

Comment:
Prior to this PR, I don't think we're handling any errors that happened if the second build attempt failed.